components/sidebar/sidebar_category/sidebar_category.test.tsx

```diff
@@ -40,6 +40,7 @@ describe('components/sidebar/sidebar_category', () => {
         setChannelRef: jest.fn(),
         getChannelRef: jest.fn(),
         handleOpenMoreDirectChannelsModal: jest.fn(),
+        isNewCategory: false,
```

Comment:
Can we add some tests so that we know the right parts of the component are rendered when in this mode?

Reply:
I haven't gotten to unit tests for this part yet, they are scheduled for later. But I can add this one now.